lib/health_checks/rails.rb

```diff
@@ -16,17 +19,39 @@ def rails
 
   private
 
-  class RedisCheck < OkComputer::Check
-    def check
-      sidekiq_response = ::Sidekiq.redis(&:ping)
+  module Logger
+    module_function
```

Comment:
Same as above

Reply:
probably a class with a static function would be better here, otherwise if I use a regular module I have to import the it to every class(3 in total) and it will add more code without adding much meaning?

Reply:
Don't understand you here, you should delete the module anyway